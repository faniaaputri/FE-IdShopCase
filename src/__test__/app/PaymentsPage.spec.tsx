import PaymentsPage from "@/app/(customer)/account/payments/page";
import Home from "@/app/page";
import { render } from "@testing-library/react";

describe("PaymentsPage", () => {
  it("should render", () => {
    const page = render(<PaymentsPage/>);
    expect(page).toMatchSnapshot();
  });
});
