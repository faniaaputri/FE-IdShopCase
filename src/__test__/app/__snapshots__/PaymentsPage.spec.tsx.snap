// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`PaymentsPage should render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="w-full h-5/6  overflow-auto"
      >
        <div
          class="group/field-group @container/field-group flex w-full flex-col gap-7 data-[slot=checkbox-group]:gap-3 [&>[data-slot=field-group]]:gap-4"
          data-slot="field-group"
        >
          <div
            aria-required="false"
            dir="ltr"
            role="radiogroup"
            style="outline: none;"
            tabindex="0"
          >
            <label
              class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
              data-slot="field-label"
              for="qris"
            >
              <div
                class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
                data-orientation="horizontal"
                data-slot="field"
                role="group"
              >
                <img
                  alt="payment-icon"
                  data-nimg="1"
                  decoding="async"
                  height="80"
                  loading="lazy"
                  src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                  style="color: transparent;"
                  width="80"
                />
                <button
                  aria-checked="true"
                  class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                  data-radix-collection-item=""
                  data-slot="radio-group-item"
                  data-state="checked"
                  id="qris"
                  role="radio"
                  tabindex="-1"
                  type="button"
                  value="qris"
                >
                  <span
                    class="relative flex items-center justify-center"
                    data-slot="radio-group-indicator"
                    data-state="checked"
                  >
                    <svg
                      aria-hidden="true"
                      class="lucide lucide-circle fill-foreground absolute top-1/2 left-1/2 size-3 -translate-x-1/2 -translate-y-1/2"
                      fill="none"
                      height="24"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                      viewBox="0 0 24 24"
                      width="24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <circle
                        cx="12"
                        cy="12"
                        r="10"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </label>
            <label
              class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
              data-slot="field-label"
              for="dana"
            >
              <div
                class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
                data-orientation="horizontal"
                data-slot="field"
                role="group"
              >
                <img
                  alt="payment-icon"
                  data-nimg="1"
                  decoding="async"
                  height="80"
                  loading="lazy"
                  src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                  style="color: transparent;"
                  width="80"
                />
                <button
                  aria-checked="false"
                  class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                  data-radix-collection-item=""
                  data-slot="radio-group-item"
                  data-state="unchecked"
                  id="dana"
                  role="radio"
                  tabindex="-1"
                  type="button"
                  value="dana"
                />
              </div>
            </label>
            <label
              class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
              data-slot="field-label"
              for="gopay"
            >
              <div
                class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
                data-orientation="horizontal"
                data-slot="field"
                role="group"
              >
                <img
                  alt="payment-icon"
                  data-nimg="1"
                  decoding="async"
                  height="80"
                  loading="lazy"
                  src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                  style="color: transparent;"
                  width="80"
                />
                <button
                  aria-checked="false"
                  class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                  data-radix-collection-item=""
                  data-slot="radio-group-item"
                  data-state="unchecked"
                  id="gopay"
                  role="radio"
                  tabindex="-1"
                  type="button"
                  value="gopay"
                />
              </div>
            </label>
            <label
              class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
              data-slot="field-label"
              for="shopeepay"
            >
              <div
                class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
                data-orientation="horizontal"
                data-slot="field"
                role="group"
              >
                <img
                  alt="payment-icon"
                  data-nimg="1"
                  decoding="async"
                  height="80"
                  loading="lazy"
                  src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                  style="color: transparent;"
                  width="80"
                />
                <button
                  aria-checked="false"
                  class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                  data-radix-collection-item=""
                  data-slot="radio-group-item"
                  data-state="unchecked"
                  id="shopeepay"
                  role="radio"
                  tabindex="-1"
                  type="button"
                  value="shopeepay"
                />
              </div>
            </label>
            <label
              class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
              data-slot="field-label"
              for="cod"
            >
              <div
                class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
                data-orientation="horizontal"
                data-slot="field"
                role="group"
              >
                <img
                  alt="payment-icon"
                  data-nimg="1"
                  decoding="async"
                  height="80"
                  loading="lazy"
                  src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                  style="color: transparent;"
                  width="80"
                />
                <button
                  aria-checked="false"
                  class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                  data-radix-collection-item=""
                  data-slot="radio-group-item"
                  data-state="unchecked"
                  id="cod"
                  role="radio"
                  tabindex="-1"
                  type="button"
                  value="cod"
                />
              </div>
            </label>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="w-full h-5/6  overflow-auto"
    >
      <div
        class="group/field-group @container/field-group flex w-full flex-col gap-7 data-[slot=checkbox-group]:gap-3 [&>[data-slot=field-group]]:gap-4"
        data-slot="field-group"
      >
        <div
          aria-required="false"
          dir="ltr"
          role="radiogroup"
          style="outline: none;"
          tabindex="0"
        >
          <label
            class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
            data-slot="field-label"
            for="qris"
          >
            <div
              class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
              data-orientation="horizontal"
              data-slot="field"
              role="group"
            >
              <img
                alt="payment-icon"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
              <button
                aria-checked="true"
                class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                data-radix-collection-item=""
                data-slot="radio-group-item"
                data-state="checked"
                id="qris"
                role="radio"
                tabindex="-1"
                type="button"
                value="qris"
              >
                <span
                  class="relative flex items-center justify-center"
                  data-slot="radio-group-indicator"
                  data-state="checked"
                >
                  <svg
                    aria-hidden="true"
                    class="lucide lucide-circle fill-foreground absolute top-1/2 left-1/2 size-3 -translate-x-1/2 -translate-y-1/2"
                    fill="none"
                    height="24"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <circle
                      cx="12"
                      cy="12"
                      r="10"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </label>
          <label
            class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
            data-slot="field-label"
            for="dana"
          >
            <div
              class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
              data-orientation="horizontal"
              data-slot="field"
              role="group"
            >
              <img
                alt="payment-icon"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
              <button
                aria-checked="false"
                class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                data-radix-collection-item=""
                data-slot="radio-group-item"
                data-state="unchecked"
                id="dana"
                role="radio"
                tabindex="-1"
                type="button"
                value="dana"
              />
            </div>
          </label>
          <label
            class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
            data-slot="field-label"
            for="gopay"
          >
            <div
              class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
              data-orientation="horizontal"
              data-slot="field"
              role="group"
            >
              <img
                alt="payment-icon"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
              <button
                aria-checked="false"
                class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                data-radix-collection-item=""
                data-slot="radio-group-item"
                data-state="unchecked"
                id="gopay"
                role="radio"
                tabindex="-1"
                type="button"
                value="gopay"
              />
            </div>
          </label>
          <label
            class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
            data-slot="field-label"
            for="shopeepay"
          >
            <div
              class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
              data-orientation="horizontal"
              data-slot="field"
              role="group"
            >
              <img
                alt="payment-icon"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
              <button
                aria-checked="false"
                class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                data-radix-collection-item=""
                data-slot="radio-group-item"
                data-state="unchecked"
                id="shopeepay"
                role="radio"
                tabindex="-1"
                type="button"
                value="shopeepay"
              />
            </div>
          </label>
          <label
            class="items-center text-sm font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50 group/field-label peer/field-label flex w-fit gap-2 leading-snug group-data-[disabled=true]/field:opacity-50 has-[>[data-slot=field]]:w-full has-[>[data-slot=field]]:flex-col has-[>[data-slot=field]]:rounded-sm has-[>[data-slot=field]]:border [&>*]:data-[slot=field]:p-4 has-data-[state=checked]:bg-primary/5 has-data-[state=checked]:border-foreground dark:has-data-[state=checked]:bg-primary/10 bg-gradient-to-r from-transparent to-foreground/10 mb-3"
            data-slot="field-label"
            for="cod"
          >
            <div
              class="group/field flex w-full gap-3 data-[invalid=true]:text-destructive flex-row items-center justify-between [&>[data-slot=field-label]]:flex-auto has-[>[data-slot=field-content]]:items-start has-[>[data-slot=field-content]]:[&>[role=checkbox],[role=radio]]:mt-px"
              data-orientation="horizontal"
              data-slot="field"
              role="group"
            >
              <img
                alt="payment-icon"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.prod.website-files.com%2F674ab654de930b217c6389c1%2F6768ffe4b3a0965dc1ecc385_QRIS-logo.png&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
              <button
                aria-checked="false"
                class="border-background bg-background text-primary focus-visible:border-ring focus-visible:ring-ring/50 aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive dark:bg-input/30 aspect-square size-5 shrink-0 rounded-full border shadow-xs transition-[color,box-shadow] outline-none focus-visible:ring-[3px] disabled:cursor-not-allowed disabled:opacity-50"
                data-radix-collection-item=""
                data-slot="radio-group-item"
                data-state="unchecked"
                id="cod"
                role="radio"
                tabindex="-1"
                type="button"
                value="cod"
              />
            </div>
          </label>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
